python/mxnet/contrib/onnx/mx2onnx/_op_translations.py

```diff
@@ -2280,6 +2289,200 @@ def convert_layer_norm(node, **kwargs):
     return nodes
 
 
+def make_tensor(shape_list, shape_name, initializer, dtype='int64'):
```

Comment:
Please rename to create_tensor() or something, so it's a clear distinction between this and onnx.helper.make_tensor().

Reply:
Yeah noted